README.md

```diff
@@ -518,25 +619,15 @@ dictionary AILanguageModelCreateCoreOptions {
   unrestricted double topK;
   unrestricted double temperature;
 
-  sequence<DOMString> expectedInputLanguages;
-}
+  sequence<AILanguageModelExpectedInput> expectedInputs;
```

Comment:
Nit: I feel like `expected` is kind of redundant there. How about removing it and simply calls it `inputs` or `inputTypes`?

```js
const session = await ai.languageModel.create({
  inputs: [{ type: "audio" }, { type: "image" }],
});
```

Reply:
I think that would be too confusing with the actual prompts.

Reply:
Would `inputTypes` also be confusing?

Reply:
Yes, because then it's only talking about the types, not the types + languages. It's also not expressing that this is what the web developer expects to input later, which may or may not match the actual inputs.

Reply:
`expectedInputs` will be then ;)
Thanks to you and @Zubairrahman11 for sharing your reasoning.